src/java.base/share/classes/jdk/internal/util/OctalDigits.java

```diff
@@ -34,14 +34,20 @@
  *
  * @since 21
  */
-final class OctalDigits implements Digits {
-    @Stable
-    private static final short[] DIGITS;
-
+public final class OctalDigits implements Digits {
     /**
      * Singleton instance of OctalDigits.
      */
-    static final Digits INSTANCE = new OctalDigits();
+    public static final Digits INSTANCE = new OctalDigits();
```

Comment:
The constructor should be after the field definitions. 
Don't let some tool reformat the code and move things around.